app/utils/verified-builds.tsx

```diff
@@ -13,29 +16,90 @@ export type OsecRegistryInfo = {
     executable_hash: string;
     last_verified_at: string | null;
     repo_url: string;
-};
-
-export type CheckedOsecRegistryInfo = {
-    explorer_hash: string;
+    verify_command: string;
 };
 
 export function useVerifiedProgramRegistry({
     programId,
+    programAuthority,
     options,
 }: {
     programId: PublicKey;
+    programAuthority: PublicKey | null;
     options?: { suspense: boolean };
 }) {
-    const { data, error, isLoading } = useSWRImmutable(
+    const { url: clusterUrl } = useCluster();
+    const connection = new Connection(clusterUrl);
```

Comment:
As of right now this will fetch PDA args uploaded to devnet, but the command will only work for mainnet. 

I think the command should only be shown for mainnet, wdyt?

Reply:
Ah just showing it for mainnet is probably the easiest solution then we dont need an addtional hint right?

Reply:
Added a mainnet check